pages/authzed/concepts/authzed-materialize.mdx

```diff
@@ -1,45 +1,46 @@
 import { Callout } from 'nextra/components'
 
-# AuthZed Materialize (Early Access)
-
-AuthZed Materialize works with AuthZed Dedicated and is inspired by the Leopard index component described in the [Zanzibar paper](https://zanzibar.tech/2IoYDUFMAE:0:T).
-Much like the concept of a materialized view in relational databases, AuthZed Materialize supports SpiceDB permissions systems by precomputing permissions defined in your schema.
-
-By creating a materialized view of your permissions in a relational database, you can efficiently sort, search, and filter massive lists of authorized objects while leveraging the authorization computation capabilities of SpiceDB.
+# AuthZed Materialize
 
 <Callout type="info">
   AuthZed Materialize is available to users of AuthZed [Dedicated] as part of an early access program.
   Don't hesitate to get in touch with your AuthZed account team if you would like to participate.
 </Callout>
 
-[Dedicated]: ../guides/picking-a-product#dedicated
+AuthZed Materialize is inspired by the Leopard index component described in the [Zanzibar paper](https://zanzibar.tech/2IoYDUFMAE:0:T).
+Much like the concept of a materialized view in relational databases, AuthZed Materialize is a service that computes how permissions change after relationships
+are written, when those relationships affect a subject's membership in a permission set or a set’s permission on a specific resource.
+These permissions are then streamed to your client, so that you can store them as a denormalized table, and then do operations like searching, sorting, and filtering much more efficiently.
```

Comment:
This is one way to use it; the other is "Accelerated Queries," which I think is what we want to default users to, since it doesn't require additional work on their side.
```suggestion
These precomputed permissions can then either be used to provide faster checks and lookups through Accelerated Queries or streamed to your own application database to do operations like searching, sorting, and filtering much more efficiently.
```